.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
```

Comment:
@coderabbitai provide me these information:
1. the exhaustive list of all merged PRs for milestone 1.19.0 that has a bug label.
2. the exhaustive list of all merged PRs for milestone 1.19.0 that has a enhancement label.
3. the list of authors of each merged 1.19.0 milestone PRs

Reply:
Give me the full list for all three above

Reply:
@coderabbitai provide me the exhaustive list of all merged PRs for milestone 1.19.0 that has a enhancement or bug label.

Reply:
There should be alot more than this